app/javascript/onboarding/components/FollowTags.jsx

```diff
@@ -74,10 +101,31 @@ export class FollowTags extends Component {
           credentials: 'same-origin',
         }),
       ),
-    ).then((_) => {
-      const { next } = this.props;
-      next();
-    });
+    )
+      .then(() => {
+        if (email_digest_periodic) {
+          return fetch('/onboarding/notifications', {
+            method: 'PATCH',
+            headers: {
+              'X-CSRF-Token': csrfToken,
+              'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+              notifications: {
+                email_digest_periodic: this.state.email_digest_periodic,
+              },
+            }),
+            credentials: 'same-origin',
+          });
```

Comment:
So the form does a PATCH to `/onboarding/notifications` and a separatae POST to `/follows`, then moves to the next slide. I like this approach :+1: